### ISSUE TITLE
Unable to train DreamBooth in Colab

### ISSUE THREAD

**Issue by sudhir2016** [huggingface/autotrain-advanced#286]
I wanted to quickly try the Colab notebook provided so ran it with a few changes to save time as follows.

Training.
- training images - 3 pics of myself
- resolution 512.
- num_steps 50
- prompt - photo of a sks sudhir

Inference

from diffusers import DiffusionPipeline
import torch
prj_path = "/content/my_dreambooth_project"
model = "stabilityai/stable-diffusion-xl-base-1.0"
pipe = DiffusionPipeline.from_pretrained(
    model,
    torch_dtype=torch.float16,
)
pipe.to("cuda")
pipe.load_lora_weights(prj_path, weight_name="pytorch_lora_weights.safetensors")

prompt = "potrait Photo of a sks sudhir, in anime style"

for seed in range(5):
    generator = torch.Generator("cuda").manual_seed(seed)
    image = pipe(prompt=prompt, generator=generator,num_inference_steps=25)
    image = image.images[0]
    image.save(f"images_test.png")

However images_test.png was a portrait of some random person.

Please suggest what I am doing wrong. I can share my notebook if needed. 

Question do I need to resize the training images to 512 by 512 and do I need to name them in a certain way.

**Comment by sudhir2016** [comment]
My mistake. Working fine now.
